playbook/app/pb_kits/playbook/pb_date_year_stacked/docs/_date_year_stacked_default.jsx

```diff
@@ -4,7 +4,11 @@ import { DateYearStacked } from '../../'
 const DateYearStackedDefault = (props) => {
   return (
     <div>
-      <DateYearStacked date={new Date()} />
+      <DateYearStacked
+          align="left"
```

Comment:
Hey! Great Job!!! I believe you can remove line 8!  The default align of this kit is left, so there is no need to add it here.  The fix for the bug was on like 10 where you added `{...props}`!! Adding `{...props}` will make our docs pick up on the dark mode styles.